DeDRM_plugin/__init__.py

```diff
@@ -2,10 +2,13 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import with_statement
```

Comment:
```suggestion
```
This was used in Python 2.5 and can be safely removed.

Reply:
Ok, I didn’t want to do any more changes than necessary, but since you request it, I took the liberty of removing that line from all other `.py` files as well.